### ISSUE TITLE
Sidebar colour different to main body of page after upgrade to Obsidian 1.0.3

### ISSUE THREAD

**Issue by gabrielwhite** [kepano/obsidian-minimal#477]
### Is the bug present when using the default Obsidian theme?

Yes

### Is the bug present when snippets and plugins are disabled?

Yes

### Minimal theme version

6.1.8

### Describe the bug

After upgrading to Obsidian 1.0.3, the left sidebar is permanently a different colour to the body of the screen. Before upgrading it was the same (slightly lighter shade). See issue here:

<img width="199" alt="Screen Shot 2022-10-26 at 5 41 58 PM" src="https://user-images.githubusercontent.com/955848/198133663-9f124432-50a4-44e6-8ff0-14e68d3849dc.png">

### Debug info

SYSTEM INFO:
	Obsidian version: v1.0.3
	Installer version: v1.0.0
	Operating system: Darwin Kernel Version 21.6.0: Thu Sep 29 20:11:33 PDT 2022; root:xnu-8020.240.7~1/RELEASE_ARM64_T8110 21.6.0
	Login status: not logged in
	Insider build toggle: off
	Live preview: off
	Legacy editor: off
	Base theme: dark
	Community theme: Minimal
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 12
	Plugins enabled: 12
		1: Style Settings v0.4.12
		2: Smart Typography v1.0.18
		3: Shortcuts extender v2.2.0
		4: Recent Files v1.3.3
		5: Pandoc Plugin v0.4.1
		6: Ozan's Image in Editor Plugin v2.1.5
		7: Minimal Theme Settings v6.1.0
		8: Markdown prettifier v0.1.4
		9: LanguageTool Integration v0.3.0
		10: File Explorer Note Count v1.2.0
		11: Editor Syntax Highlight v0.1.3
		12: Advanced Tables v0.17.3

**Comment by kepano** [comment]
I am not seeing this issue on my end. Ribbon is the same color as sidebar. Can you show a screenshot of the full app? I need to understand all of the other settings you are using:

- Color scheme
- Window frame style
- Colorful frame on/off
- Text labels for primary navigation on/off

**Comment by kepano** [comment]
Are you using translucent window?

**Comment by gabrielwhite** [comment]
Yes, that fixed it! There must have been a change to Obsidian behaviour in the upgrade. Thanks for helping me troubleshoot.
